test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,420 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  getWindowHandles,
+} = require('../helpers');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Custom Token', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('can be sent from inside MetaMask', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        // create token
+        await driver.openNewPage(
+          `http://127.0.0.1:8080/?contract=${contractAddress}`,
+        );
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        await driver.findClickableElement('#deployButton');
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        // switch to popup and add token from dapp
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+        await driver.waitUntilXWindowHandles(2);
+        await driver.switchToWindow(extension);
+
+        const asset = await driver.waitForSelector({
+          css: '.list-item',
+          text: '10 TST',
+        });
+        asset.click();
+        // send token from extension
+        await driver.waitForSelector('[data-testid="eth-overview-send"]');
```

Comment:
You shouldn't need to use waitForSelector followed by clickElement, as clickElement waits until the element is located

Reply:
Thanks for the explanation, this issue occurs because we use the same selector on both the overview page and the asset overview page. Instead of `await driver.waitForSelector('[data-testid="eth-overview-send"]');` can we wait for the overview page to transition to the asset overview page e.g. wait for the balance to show 10 TST?

Reply:
Hmm, I removed it and it works now. Tested it locally and also works after removing waitForSelector.